### PR TITLE
refactor: store election definition in backend

### PR DESCRIPTION
**Pull request by eventualbuddha**
This assumes a new API that will be implemented by `module-scan` for now.

Depends on https://github.com/votingworks/module-scan/pull/73

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -76,27 +75,17 @@ const App: React.FC = () => {
...
+  }, [setElection])
 
-  const fetchElection = useCallback(
-    async () =>
-      fetchJSON<CardReadLongResponse>('/card/read_long').then((card) =>
-        JSON.parse(card.longValue)
-      ),
-    []
-  )
+  const fetchElection = useCallback(async () => {
```
Comment (benadida) [inline src/App.tsx:85]: should we use the opportunity to clarify this function as `loadElectionFromCard` or the like?

File: src/api/election.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { Election } from '@votingworks/ballot-encoder'
+import {
+  DeleteElectionResponse,
+  GetElectionResponse,
+  PutElectionResponse,
+} from '../config/types'
+import fetchJSON from '../util/fetchJSON'
+
+export async function get(): Promise<Election | undefined> {
+  return fetchJSON<GetElectionResponse>('/election')
```
Comment (benadida) [inline src/api/election.ts:10]: suggest prefixing as `/config/election` or the like. We can stuff a few different things  in`/config` over time.

Comment (eventualbuddha) [reply src/api/election.ts]: Done

File: src/App.tsx (hunk 1 above)
Comment (eventualbuddha) [reply src/App.tsx]: Done

**Pull request merged**